core/spec.md

```diff
@@ -588,20 +597,27 @@ Then the existing entity can be deleted.
 - Constraints:
   - MUST be a [RFC3339](https://tools.ietf.org/html/rfc3339) timestamp
   - MUST be a read-only attribute in API view
+  - If this feature is enabled after an entity has already been created,
+    that entity's `createdon` value MUST remain empty
```

Comment:
We were trying to align with https://www.apicur.io/registry/docs/apicurio-registry/2.4.x/assets-attachments/registry-rest-api.htm#tag/Metadata/operation/getArtifactMetaData
but TBH it has been bugging me recently because I find it hard to say "created on" instead of "created at" or "create time".  I've added this to agenda for today.  Not too late to change.